README.md

```diff
@@ -339,7 +339,8 @@ Statics parameters
 - `browserifyBundles`: Declare [browserify](http://browserify.org) bundles: one or more files in your `jsPath` for browserify to bundle via its [bundle method](https://github.com/substack/node-browserify#browserifyfiles--opts). Use of browserify is optional. If no bundles are defined here, the browserify step will be skipped.
   - Default: `[]`
   - Example declaring one bundle: `[{outputFile: "bundle.js", files: ["landingPage.js", "main.js", "etc.js"], params: {someOpt: "someValue"}}]`
-  - Example declaring one bundle only used in dev mode `[{outputFile: "bundle.js", context: "dev", files: ["landingPage.js", "main.js", "etc.js"], params: {someOpt: "someValue"}}]`
+  - Example declaring one bundle only used in dev mode `[{outputFile: "bundle.js", env: "dev", files: ["landingPage.js", "main.js", "etc.js"], params: {someOpt: "someValue"}}]`
+    - Omitting `env` will result in bundling in both modes.
```

Comment:
move this down to where the other `Note:` is, place it above the `params` note and follow the same formatting: "Note: Omitting `env` will..."